invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,92 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2019 CERN.
+// Copyright (C) 2019 Northwestern University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Container, Grid } from "semantic-ui-react";
+import _truncate from "lodash/truncate";
```

Comment:
discussion: if we only need `truncate`, my vote would be to implement it ourselves rather than bring lodash in, unless we do plan on using other lodash functions. Just want us to be careful about accruing dependencies and load size.

Reply:
This import will only include the `truncate` function and the whole lodash library. On top of that lodash is coming already with searchkit :) But I agree to be careful with the dependencies we are using :)

Reply:
If react-searchkit already uses it that's fine :)